egs/wsj/s5/steps/nnet3/tdnn/make_configs.py

```diff
@@ -28,6 +28,9 @@ def GetArgs():
                             help="Raw feature dimension, e.g. 13")
     feat_group.add_argument("--feat-dir", type=str,
                             help="Feature directory, from which we derive the feat-dim")
+    parser.add_argument("--num-cmn-offsets", type=int, 
```

Comment:
these changes also won't be necessary once you upgrade to xconfig scripts [I just hand-specify the iVector dim there, it's not worked out from any directory.]